YapDatabase/Extensions/View/YapDatabaseViewTransaction.m

```diff
@@ -1308,6 +1308,13 @@ - (void)insertRowid:(int64_t)rowid collectionKey:(YapCollectionKey *)collectionK
 		
 		NSString *pageKey = pageMetadata->pageKey;
 		YapDatabaseViewPage *page = [self pageForPageKey:pageKey];
+        
+        NSAssert(page != nil, @"Missing page in group(%@)", group);
```

Comment:
Any idea what can lead to this?

FWIW we've seen similar crashes, but haven't root-caused it.

Hopefully this assert/logging will help track it down...

Reply:
Not sure, I've never caught it live, only from crash logs